source/_integrations/zimi.markdown

```diff
@@ -0,0 +1,125 @@
+2. Zimi devices and names will be mapped per HA guidelines in the table below. The user may change these names to more friendly names - see [Customizing entities](/docs/configuration/customizing-devices/).
+
+When you add a supported device, the following entities will be created:
+
+| Zimi product                    | HA device name | HA entities         | HA default friendly name                                         | Supported |
+|---------------------------------|----------------|---------------------|------------------------------------------------------------------|-----------|
+| Blind Controller                | Cover          | 1xCover             | Cover {Name}                                                     | Future    |
+| Fan and Light Controller        | Fan            | 1xFan<br>1xSwitch   | Fan {Name}<br>Fan {Name}                                         | Future    |
+| Garage Door Controller          | Cover          | 1xCover<br>2xSensor | Cover {Name}<br>Cover {Temperature}<br>Cover {Humidity}          | Future    |
+| Light Dimmer Switch             | Light          | 1xLight             | Light {Name}                                                     | Yes       |
+| Multi Dimmer Switch (2 button)  | Light          | 1xLight             | Light {Name}                                                     | Yes       |
+| Multi Dimmer Switch (4 button)  | Light          | 2xLight             | Light {Name}<br>Light {Name}                                     | Yes       |
+| Multi-Purpose Switch (1 button) | Switch         | 1xSwitch            | Switch {Name}                                                    | Future    |
+| Multi-Purpose Switch (2 button) | Switch         | 2xSwitch            | Switch {Name}<br>Switch {Name}                                   | Future    |
+| Multi-Purpose Switch (3 button) | Switch         | 3xSwitch            | Switch {Name}<br>Switch {Name}<br>Switch {Name}                  | Future    |
+| Multi-Purpose Switch (4 button) | Switch         | 4xSwitch            | Switch {Name}<br>Switch {Name}<br>Switch {Name}<br>Switch {Name} | Future    |
+|Power Point                      | Outlet         | 2xOutlet            | Outlet {Name}|Future|
+
+### Zimi light controller
+
+- Light entity: Basic on/off and brightness control
+
+## Data updates
+
+The integration receives updates instantly from the Zimi Cloud Controller via the Zimi API.
+
+## Known limitations
+
+1. Entity name changes made in the Zimi app will not be reflected in Home Assistant until after a restart. This is because entity names are only read during integration setup and Home Assistant startup.
+
+
+## Troubleshooting
+
+### Missing Zimi Devices
+
+If there are missing Zimi devices after the initial integration, you may have to run the discovery process again.
+
+To do this:
+
+1. Go to **Settings** > **Devices & Services**
+2. Click on **Zimi**
+3. Click **Add Hub**
```

Comment:
```suggestion
2. Select **Zimi**
3. Select **Add Hub**
```
we [avoid click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)